### PR TITLE
loop: v0.8.1 -> v0.9.0

### PR DESCRIPTION
**Pull request by nixbitcoin**
This PR updates the loop pkg. It also updates the module package to fully utilize the changes introduced in v0.9.0.

When we release this we need to include instructions, ideally a one-line command, to migrate existing loop directories from `/var/lib/lnd/.loop/ to `/var/lib/lightning-loop`.

### CODE REVIEW

**Review by jonasnick — APPROVED** [review]
ACK 56705fbd16c98e2b9f3f1098a6c994f19a97f47b

**Review by erikarvstedt — CHANGES_REQUESTED** [review]
[Fixups.](https://github.com/erikarvstedt/nix-bitcoin/commits/loop-0.9.0-ea)

File: modules/lightning-loop.nix [hunk 1]
```diff
@@ -47,27 +69,33 @@ in {
...
       serviceConfig = nix-bitcoin-services.defaultHardening // {
-        ExecStart = ''
-          ${cfg.package}/bin/loopd \
-          --lnd.host=${config.services.lnd.listen}:10009 \
-          --lnd.macaroondir=${config.services.lnd.dataDir}/chain/bitcoin/mainnet \
-          --lnd.tlspath=${secretsDir}/lnd-cert \
-          ${optionalString (cfg.proxy != null) "--server.proxy=${cfg.proxy}"} \
-          ${cfg.extraArgs}
-        '';
+        WorkingDirectory = cfg.dataDir;
```
Comment (erikarvstedt) [inline modules/lightning-loop.nix:85]: After a bit of testing it seems `WorkingDirectory` isn't used.

**Comment by nixbitcoin** [comment]
Rebased, squashed your fixups and removed `WorkingDirectory`

**Review by erikarvstedt — APPROVED** [review]
ACK.
We should merge [BTCPayServer from upstream & postgresqlBackup](https://github.com/fort-nix/nix-bitcoin/pull/237) first

**Comment by jonasnick** [comment]
reACK 56705fbd16c98e2b9f3f1098a6c994f19a97f47b

**Comment by erikarvstedt** [comment]
Commit `test: increase diskSize` was included in 73f4275d2a2bdef34644aabbefa43ae748fd6d6f and can be removed.

**Comment by erikarvstedt** [comment]
ACK a89a3e934f485bd9b0c159e18f6af91883c56e84

**Comment by jonasnick** [comment]
> Commit `test: increase diskSize` was included in 73f4275d2a2bdef34644aabbefa43ae748fd6d6f and can be removed. 

@erikarvstedt That's what I was thinking too immediately after I acked again, but you apparently removed this comment, why?

**Comment by erikarvstedt** [comment]
Yeah, removing the commit improves the git history, but it doesn't affect the final code.
I'd have preferred to just get done with this PR. :wink:
I think it's okay if you would just merge 24b506ff8a2c2d788e102548423cb75bfe4d81e4, which is the direct parent of the ACK'd a89a3e934f485bd9b0c159e18f6af91883c56e84, and removing a89a3e934f485bd9b0c159e18f6af91883c56e84 itself is totally uncontroversial.

ACK https://github.com/fort-nix/nix-bitcoin/commit/24b506ff8a2c2d788e102548423cb75bfe4d81e4

**Comment by jonasnick** [comment]
For posteriority I merged a89a3e934f485bd9b0c159e18f6af91883c56e84 because it wasn't straightforward to remove within my merge script.

**Pull request merged**